user/docs/faq.md

```diff
@@ -0,0 +1,157 @@
+  </TabItem>
+</Tabs>
+
+### I am getting an error saying "Could not find or load main class @user_jvm_args.txt", what do I do?
+
+There are two common reasons for this:
+
+- You are running an outdated version of Java. Please see the [Installing Java][installjava] section.
+- You have multiple versions of Java installed on your computer, possibly installed one of them just right now, but your computer is still using the old one. This means that your PATH variable is not set correctly. Please follow the steps outlined [here][wrongjava].
+
+## Playing NeoForge
+
+Sometimes, issues show up not during installation, but during gameplay.
+
+### My game is lagging! What can I do?
+
+Identify whether the problem is on the server or the client (or both).
+
+- If your game is running at low FPS (frames per second), please read [My client is lagging! What can I do?][clientlag]
+- If your game is running at decent enough FPS, but your world is lagging (machines running too slowly, mobs lagging behind visually, etc.), please read [My server is lagging! What can I do?][serverlag]
+
+### My server is lagging! What can I do?
+
+There is multiple reasons why this could be. A common offender are too many entities (= mobs or dropped items) or block entities (= chests, machines, etc.) being nearby.
+
+If you want an exact breakdown of what is lagging, try using the [Spark][spark] mod. Spark is a profiler that can tell you what code paths are taking how long. To use it:
+
+- Join a world.
+- Run the `/spark profiler` command.
+- Wait a couple of minutes.
+- Run the `/spark profiler --stop` command.
+- Open the link and have a look or, if you can't make any sense of it, post it in the [Discord server][support]'s `#user-support` channel and we will try our best to help you.
+
+### My client is lagging! What can I do?
+
+This is commonly because Minecraft is not a very optimized game, and mods can make this worse. If you are running shaders, chances are also that your graphics card is not strong enough to support them.
+
+If you want an exact breakdown of what is lagging, try using the [Spark][spark] mod. Spark is a profiler that can tell you what code paths are taking how long. To use it:
+
+- Join a world.
+- Run the `/sparkc profiler` command. (Note the use of `/sparkc`, as opposed to `/spark` to profile the server.)
+- Wait a couple of minutes.
+- Run the `/sparkc profiler --stop` command.
+- Open the link and have a look or, if you can't make any sense of it, post it in the [Discord server][support]'s `#user-support` channel and we will try our best to help you.
+
+### How do I find a faulty mod?
+
+If you need to find a faulty mod without any lead on where to start, your best bet may be to use a binary search. The binary search is a common way to find a faulty things among many, without going through things one by one. Reworded for mods, it goes as follows:
+
+1. Remove half of the existing mods and put them into a different folder.
```

Comment:
Maybe it should be mentioned that this may cause new errors if you accidentally remove dependencies of mods lower down